.github/workflows/publish.yml

```diff
@@ -43,6 +43,10 @@ jobs:
       anaconda_package: astropy
       anaconda_keep_n_latest: 10
 
+      env: |
```

Comment:
I think these two lines need a comment explaining what they do. Is the idea that from now on all our nightly wheels are going to be build against numpy dev?

Reply:
Thanks!